include/rocksdb/advanced_options.h

```diff
@@ -294,10 +294,10 @@ struct AdvancedColumnFamilyOptions {
   // The maximum number of write buffers that are built up in memory.
   // The default and the minimum number is 2, so that when 1 write buffer
   // is being flushed to storage, new writes can continue to the other
-  // write buffer.
-  // If max_write_buffer_number > 3, writing will be slowed down to
-  // options.delayed_write_rate if we are writing to the last write buffer
-  // allowed.
+  // write buffer without triggering write delay.
+  // This parameter affects write delay condition. It does not affect the
+  // timing of write buffer flush. When the number of write buffers exceeds
```

Comment:
"When the number of write buffers exceeds... " is not exactly correct. See https://github.com/facebook/rocksdb/blob/01f2edd145e318228936b89834d058060ed600c6/db/column_family.cc#L937

I am also debating whether we should expose that internal detail in API comment. 

Should we just say "this parameter affects write delay and stop condition" with something like "the higher this option value is, the more write buffers need to be accumulated before triggering write delay (at rate of `options.delayed_write_rate`) or write stop"